azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -263,6 +263,12 @@ message InvocationRequest {
 
   // Populates activityId, tracestate and tags from host
   RpcTraceContext trace_context = 5;
+
+  // current retry count
+  int32 retry_count = 6;
```

Comment:
The default value for retry_count and max_retry_count will be 0 (because it is a [proto3 numeric type](https://developers.google.com/protocol-buffers/docs/proto3#default)). So even if it isn't explicitly set, the customer will see `context.executionContext.maxRetryCount` => 0 and `context.executionContext.retryCount` => 0. 

I think this is ok and desirable behavior but I just want to double check!

Reply:
I wrapped retry properties into `RetryContext`. Exactly the same we had in C# `ExecutionContext`.